src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.cpp

```diff
@@ -0,0 +1,366 @@
+    noexcept {
+  return {make_with_value<tnsr::i<DataType, 3, Frame::Inertial>>(x, 0.)};
+}
+
+// Conformal factor
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<Xcts::Tags::ConformalFactor<DataType>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<Xcts::Tags::ConformalFactor<DataType>> /*meta*/) const noexcept {
+  const DataType r = get(magnitude(x));
+  return {Scalar<DataType>{1. + 0.5 / r}};
+}
+
+// Conformal factor gradient
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<::Tags::deriv<Xcts::Tags::ConformalFactor<DataType>,
+                                  tmpl::size_t<3>, Frame::Inertial>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<::Tags::deriv<Xcts::Tags::ConformalFactor<DataType>,
+                             tmpl::size_t<3>, Frame::Inertial>> /*meta*/) const
+    noexcept {
+  const DataType r = get(magnitude(x));
+  const DataType isotropic_prefactor = -0.5 / cube(r);
+  auto conformal_factor_gradient =
+      make_with_value<tnsr::i<DataType, 3, Frame::Inertial>>(r, 0.);
+  get<0>(conformal_factor_gradient) = isotropic_prefactor * get<0>(x);
+  get<1>(conformal_factor_gradient) = isotropic_prefactor * get<1>(x);
+  get<2>(conformal_factor_gradient) = isotropic_prefactor * get<2>(x);
+  return {std::move(conformal_factor_gradient)};
+}
+
+// Lapse (times conformal factor)
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<Xcts::Tags::LapseTimesConformalFactor<DataType>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<Xcts::Tags::LapseTimesConformalFactor<DataType>> /*meta*/) const
+    noexcept {
+  const DataType r = get(magnitude(x));
+  return {Scalar<DataType>{1. - 0.5 / r}};
+}
+
+// Lapse (times conformal factor) gradient
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<
+    ::Tags::deriv<Xcts::Tags::LapseTimesConformalFactor<DataType>,
+                  tmpl::size_t<3>, Frame::Inertial>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<::Tags::deriv<Xcts::Tags::LapseTimesConformalFactor<DataType>,
+                             tmpl::size_t<3>, Frame::Inertial>> /*meta*/) const
+    noexcept {
+  const DataType r = get(magnitude(x));
+  auto conformal_factor_gradient =
+      make_with_value<tnsr::i<DataType, 3, Frame::Inertial>>(r, 0.);
+  const DataType isotropic_prefactor = 0.5 / cube(r);
+  get<0>(conformal_factor_gradient) = isotropic_prefactor * get<0>(x);
+  get<1>(conformal_factor_gradient) = isotropic_prefactor * get<1>(x);
+  get<2>(conformal_factor_gradient) = isotropic_prefactor * get<2>(x);
+  return {std::move(conformal_factor_gradient)};
```

Comment:
In these coordinates, grad(LapseTimesConformalFactor) == -1. * grad(ConformalFactor). Consider reducing code duplication by having one just call and then negate the other?